ui/app/AppLayouts/Chat/ChatColumn/ChatComponents/ChatCommandModal.qml

```diff
@@ -110,6 +121,7 @@ ModalPopup {
                 toAccount: root.isRequested ? selectFromAccount.selectedAccount : selectRecipient.selectedRecipient
                 asset: txtAmount.selectedAsset
                 amount: { "value": txtAmount.selectedAmount, "fiatValue": txtAmount.selectedFiatAmount }
+                toValid: addressRequestedValidator.isAddressRequested
```

Comment:
@emizzle I assume there was a bug in the code you've suggested...

We have to ensure `toValid` is `false` otherwise the warn icon won't render. `isAddressRequested` will be false when `isRequested == false`. And `isRequested` is `false` when we do a `sendTransaction` command, which is exactly the scenario in which we want the warning to show up.

Reply:
Also as stated in the other comments, we can omit the validator ceremony and simply pass `root.isRequested` here directly..